src/runtime_src/driver/xclng/tools/xbutil/xbutil.h

```diff
@@ -722,6 +775,239 @@ class device {
 
         lines.push_back(ss.str());
     }
+    
+    int readSensors( void ) const
```

Comment:
We are not reading just sensor output here. Maybe we should split the function into two: one is for reading static info and the other is for dynamic info. We may need to call dynamic one in xbutil top repeatedly to get up-to-date info.